ansible/roles/contiv/files/contiv_cni.conf

```diff
@@ -0,0 +1,5 @@
+{
+  "cniVersion": "0.1.0",
```

Comment:
can u parameterize the chi version instead of hard coding. Throw the param/value in defaults/main